src/utils/dataFormatters/dataFormatters.ts

```diff
@@ -135,14 +135,10 @@ export const formatCPUWithLabel = (value?: number) => {
     return `${localizedCores} ${i18n('format-cpu.cores', {count: cores})}`;
 };
 
-export const formatDateTime = (value?: number | string, defaultValue = 'N/A') => {
-    if (!isNumeric(value)) {
-        return '';
-    }
-
+export const formatDateTime = (value?: number | string, defaultValue = '') => {
     const formattedData = dateTimeParse(Number(value))?.format('YYYY-MM-DD HH:mm');
 
-    return Number(value) > 0 && formattedData ? formattedData : defaultValue;
+    return formattedData ?? defaultValue;
 };
```

Comment:
Do not display `N/A` for `undefined` fields by default. In case below we just don't display date fields:
![Screenshot 2024-07-23 at 15 18 10](https://github.com/user-attachments/assets/034825ff-cf00-43c3-b0c3-651dc7a9fa6c)